docs/diagrams/SortParsingSequenceDiagram.puml

```diff
@@ -25,19 +25,19 @@ deactivate SortCommandParser
 AddressBookParser -> SortCommandParser : parse("properties price asc")
 activate SortCommandParser
 
-SortCommandParser -> ParserUtil : parseActor(str)
+SortCommandParser -> ParserUtil : parseActor(str, 0)
 activate ParserUtil
 
 ParserUtil --> SortCommandParser
 deactivate ParserUtil
 
-SortCommandParser -> ParserUtil : parseSortType(str)
+SortCommandParser -> ParserUtil : parseSortType(str, 1)
 activate ParserUtil
 
 ParserUtil --> SortCommandParser
 deactivate ParserUtil
 
-SortCommandParser -> ParserUtil : parseSortDirection(str)
+SortCommandParser -> ParserUtil : parseSortDirection(str, 2)
 activate ParserUtil
 
 ParserUtil --> SortCommandParser
```

Comment:
The above changes are alright but maybe the 's' object can be given a more meaningful name?
![image](https://user-images.githubusercontent.com/23572486/140352136-b959cffa-0a5b-43a0-b1ec-98515e50935f.png)